peps/pep-0774.rst

```diff
@@ -1,6 +1,7 @@
 PEP: 774
 Title: Removing the LLVM requirement for JIT builds
 Author: Savannah Ostrowski <savannah@python.org>
+Discussions-To: https://discuss.python.org/t/pep-774-removing-the-llvm-requirement-for-jit-builds/78548
```

Comment:
You could use the slightly shorter link here, and add a trailing comma to reduce the diff for any future threads (the trailing comma is removed on render)
```suggestion
Discussions-To: https://discuss.python.org/t/78548/,
```

Reply:
@savannahostrowski Oops, sorry! I got mixed up with Post-History, which does accept trailing commas -- unlike Discussions-To.

To that point, if you make any further PRs updating the PEP, please could you add Post-History? It's probably not worth a standalone PR though.